test/extensions/filters/http/common/fuzz/filter_corpus/crash-bb74d7280823776808e881b20c0a9c87f7a2163b

```diff
@@ -0,0 +1,16 @@
+config {
+  name: "envoy.grpc_json_transcoder"
+  typed_config {
+    type_url: "type.googleapis.com/envoy.extensions.filters.http.grpc_json_transcoder.v3.GrpcJsonTranscoder"
+    value: "\n\001%8\001"
```

Comment:
This can't be the full descriptor, right? I guess it doesn't matter because the fuzzer will set the field, but it's a little confusing

Reply:
Yeah, the corpus entries will add the bookstore.Bookstore service and a proto_descriptor_bin added. This corpus entry caught a debug assert in parsing file descriptors, but it's not necessary to keep now. Will update description to add info about the corpus entries.